controller/internal/supervisor/iptablesctrl/rules.go

```diff
@@ -18,6 +20,10 @@ var triremChains = `
 `
 
 var globalRules = `
+{{if isLocalServer}}
+{{.MangleTable}} {{.MainNetChain}} -m connmark --mark 0x600  -j {{.SelfNetChain}}
```

Comment:
Let us not add magic numbers here. It is much easier to read iptables rules currently as all the marks have context with it eg. {{.RawSocketMark}} {{.DefaultConnmark}}